mgmt_upgrade_test.py

```diff
@@ -40,8 +42,23 @@ def _create_and_add_cluster(self):
     def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
         target_upgrade_server_version = self.params.get('target_scylla_mgmt_server_repo')
         target_upgrade_agent_version = self.params.get('target_scylla_mgmt_agent_repo')
-        manager_node = self.monitors.nodes[0]
+        node_idx = 0
+        manager_node: MonitorSetAWS = self.monitors.nodes[node_idx]
         manager_tool = get_scylla_manager_tool(manager_node=manager_node)
+
+        # noinspection PyBroadException
+        try:
+            LOGGER.warning("CLUSTER TYPE {}\nNode type{}".format(
```

Comment:
node is a node, it doesn't have nodes under it.
db_cluster/monitors/loaders(cluster/set) has nodes